packages/node_modules/@node-red/editor-client/locales/en-US/editor.json

```
@@ -647,7 +647,7 @@
             "tab-nodes": "Nodes",
             "tab-install": "Install",
             "sort": "sort:",
-            "sortRelevance": "relevance",
+            "sortDownloads": "downloads",
```

```suggestion
            "sortRelevance": "relevance",
```

Keep the existing label (also avoids translation churn)